non-channel n values

jacob-a-brown opened the pull request:
Allows the user to edit the n-values outside of the channel

jacob-a-brown committed:
Added get_culvert method

Added a get_culvert method to return a Culvert object of a given id

jacob-a-brown committed:
Update prg.py

jacob-a-brown committed:
non-channel n values

Added "alter_nonchannel_n" to allow the user to alter n-values outside of the channel.

jacob-a-brown committed:
Update cross_section.py

jacob-a-brown committed:
Update cross_section.py

mikebannis committed:
Merge branch 'master' into master

mikebannis commented on parserasgeo/features/cross_section.py:
```diff
@@ -574,35 +574,29 @@ def alter_channel_n(self, scalar):
-
+        if self.channel_n is None:
+            raise ChannelNError('The channel is undefined. Run define_channel_n before using alter_nonchannel_n')
+        else:
```
Please delete this else in the interest of flatter code, and shift everything from new line 591 to 599 left 4 spaces. Same functionality but easier to read.

mikebannis commented on parserasgeo/features/cross_section.py:
```diff
@@ -558,11 +556,13 @@ def alter_channel_n(self, scalar):
-        if self.channel_n is not None:
+        if self.channel_n is None:
+            raise ChannelNError('The channel is undefined. Run define_channel_n before using alter_channel_n')
+        else:
```
Please remove the else.

jacob-a-brown committed:
Update cross_section.py

Removed ability to move bank stations (never did it) and associated error. Added ability to alter the non-channel n values. Addressed comments

jacob-a-brown committed:
Merge pull request #2 from pasteurizedcows/JAB---non-channel-n

Update cross_section.py

jacob-a-brown committed:
Merge pull request #1 from pasteurizedcows/JAB---get_culvert

Added get_culvert method

mikebannis commented:
Commit is based on older (python 2 only) version of prg. Let's discuss on Monday
